4-css-grid/styles.css

```diff
@@ -0,0 +1,18 @@
+* section {
+    background-color: rgb(66, 66, 66);
+    border-radius: 15px;
+    color: whitesmoke;
+    margin: 5px;
+    padding: 20px;
+}
+body {
+    display: grid;
```

Comment:
don't apply this directly to the body, use a wrapper. If you do this and you want to include another element which is not a grid, you will have to deal with display issues.